src/autopas/containers/verletListsCellBased/verletListsCells/traversals/VLCSlicedC02Traversal.h

```diff
@@ -46,32 +41,36 @@ class VLCSlicedC02Traversal
    * @param interactionLength cutoff + skin
    * @param cellLength length of the underlying cells
    * @param pairwiseFunctor The functor that defines the interaction of two particles.
+   * @param dataLayout
+   * @param useNewton3
+   * @param typeOfList indicates the type of neighbor list as an enum value, currently only used for getTraversalType
    */
   explicit VLCSlicedC02Traversal(const std::array<unsigned long, 3> &dims, PairwiseFunctor *pairwiseFunctor,
-                                 double interactionLength, const std::array<double, 3> &cellLength)
-      : SlicedC02BasedTraversal<ParticleCell, PairwiseFunctor, dataLayout, useNewton3, false>(
-            dims, pairwiseFunctor, interactionLength, cellLength),
+                                 double interactionLength, const std::array<double, 3> &cellLength,
+                                 DataLayoutOption dataLayout, bool useNewton3, ContainerOption::Value typeOfList)
+      : SlicedC02BasedTraversal<ParticleCell, PairwiseFunctor>(dims, pairwiseFunctor, interactionLength, cellLength,
+                                                               dataLayout, useNewton3, false),
+        VLCTraversalInterface<typename ParticleCell::ParticleType, NeighborList>(typeOfList),
         _functor(pairwiseFunctor) {}
 
   void traverseParticlePairs() override;
 
-  [[nodiscard]] DataLayoutOption getDataLayout() const override { return dataLayout; }
-
-  [[nodiscard]] bool getUseNewton3() const override { return useNewton3; }
-
   [[nodiscard]] TraversalOption getTraversalType() const override {
-    switch (typeOfList) {
+    switch (this->_typeOfList) {
       case (ContainerOption::verletListsCells):
         return TraversalOption::vlc_sliced_c02;
       case (ContainerOption::pairwiseVerletLists):
         return TraversalOption::vlp_sliced_c02;
+      default:
+        autopas::utils::ExceptionHandler::exception("Traversal was created with an unsupported neighborlist type: " +
+                                                    ContainerOption::getOptionNames()[this->_typeOfList]);
```

Comment:
```suggestion
        autopas::utils::ExceptionHandler::exception("Traversal was created with an unsupported neighborlist type: {}",
                                                    this->_typeOfList.to_string());
```
The array access would result in a segfault if the value is weird. `to_string` handles this well.